sdk/docs/manually-written/overview/overview/multiparty-application-intro.rst

```diff
@@ -3,180 +3,130 @@
+using the Daml smart contract language and synchronizing the ledger state using the Canton protocol.
+
+The Daml language is a smart contract language for multi-party applications. Conceptually, Daml is similar to the Structured Query Language (SQL) used in traditional database systems, describing the data schema and rules for manipulating the data.
+
+The Daml language:
+ - defines the shared state between the parties, including process permissions and data ownership
+ - defines workflows, execution policies, and read/write permissions
+ - enables developers to build rich transactions that codify strict business rules
+ - defines the APIs through which multi-party applications can talk to each other and compose
+
+The Daml code that collectively makes up the data schema and rules for an application is called a Daml model. Increasingly sophisticated and valuable solutions are composed from existing Daml models, enabling a rich ecosystem that accelerates application development.
+
+Using the Daml language, developers define the schema for a shared ledger. The Canton protocol ensures that each party gets a unique view into the shared ledger, which is their projection of the full system.
+
+In the execution model for Canton, each party of the application is hosted on a Participant Node (Diagram 1). The Participant Node stores the party’s unique projection and history of the shared system of record. Participant Nodes synchronize by running a consensus protocol (the Canton Protocol) between them. The protocol is executed by sending encrypted messages through synchronizers which route messages and offer guaranteed delivery and order consistency. Synchronizers are also units of access control and availability, meaning an application can be additionally protected from interference by other applications or malicious actors by synchronizing it only through a given synchronizers, and restricting which participants can connect to it.
+
+Diagram 1:
+
+.. figure:: arch-intro-1.png
+   :alt: A synchronizer (center) with four participant nodes. Participant Node One hosts Party A; Participant Node Two hosts Party B; Participant Node Three hosts Party C; and Participant Node Four hosts Parties D, E, and F. The synchronizer can be centralized or distributed, public or private.
+
+
+Transfer Example in the Canton Network
+--------------------------------------
+
+Consider the transfer example described above with Alice and Bob. Using Daml, the process looks like this:
+
+#. Alice logs into her online banking at Bank A and enters a transfer to Bob at Bank B.
+#. The online banking backend creates a transaction that deducts $100 from Alice’s account and creates a transfer to Bob at Bank B.
+#. When Bank B accepts the transfer, Bank A credits $100 to Bank B’s account at Bank A and Bank B simultaneously credits Bob’s account by $100.
+#. Bob’s online banking interfaces with the Canton Ledger and can see the incoming funds in real time.
```

Comment:
This still sounds like a multi-step process (the money is first deducted and moved into a transfer, then credited to bank B and then internally credited at bank B). It would be good to point out why this is faster now than the traditional processes where each step takes a day or so.